Documentation/source/userguide/userguide.md

```diff
@@ -627,14 +627,14 @@ number_of_runs_in_parallel = 25
 >
 > description: The number of configuration runs that can run in parallel. 
 
-`clis_per_node`
+`max_parallel_runs_per_node`
 > aliases: N/A
```

Comment:
Keep clis_per_node for backwards compatibility (see ticket). Use max_parallel_runs_per_node as default notation and clis_per_node as alias where necessary/possible. So documentation wise we will refer to it in general as max_parallel but the user can access it in sparkle with the clis_per_node notation.